### ISSUE TITLE
jsonexcetion jsonvalue is not an object

### ISSUE THREAD

**Issue by webe3** [abraunegg/onedrive#368]
I had onedrive synchronize working last week.  Now I am getting a failure when running from the command line.
```
$ onedrive --synchronize
Initializing the Synchronization Engine ...
std.json.JSONException@std/json.d(274): JSONValue is not an object
----------------
??:? pure @safe void std.exception.bailOut!(std.json.JSONException).bailOut(immutable(char)[], ulong, scope const(char)[]) [0x55c594946309]
??:? pure @safe bool std.exception.enforce!(std.json.JSONException).enforce!(bool).enforce(bool, lazy const(char)[], immutable(char)[], ulong) [0x55c5948da981]
??:? inout pure @property @trusted inout(std.json.JSONValue[immutable(char)[]]) std.json.JSONValue.objectNoRef() [0x7fc8a00a5cdd]
??:? [0x55c5948bda10]
??:? [0x55c5948bfb0c]
??:? [0x55c5948def72]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).runAll() [0x7fc89fdba12f]
??:? _d_run_main [0x7fc89fdb9f45]
??:? __libc_start_main [0x7fc89f99909a]
??:? [0x55c5948acbf9]
```

**Comment by abraunegg** [comment]
@webe3 
Please can you amend / edit your issue post above with the information as per https://github.com/abraunegg/onedrive/blob/master/README.md#extra

**Comment by webe3** [comment]
I have fixed the problem.  My home directory had filled up due to a large x2go session log.  After I cleaned that out and rebooted I had the onedrive issue.  I suspected the problem was caused by the filled directory and left onedrive in an odd state.  In my ~/.config/onedrive directory, I moved the following files out of the way: items.sqlite3 refresh_token and resume_upload.  I manually ran the synchronize and had to go through the authorize step again.  Once I did that, the synchronize worked.  Sorry for the trouble.  Thanks for the application.
